compiler/test/fail_compilation/issue3396.d

```diff
@@ -0,0 +1,25 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/issue3396.d(17): Error: call to unimplemented abstract function `void M()`
+fail_compilation/issue3396.d(17):        declared here: fail_compilation/issue3396.d(12)
+---
+*/
+module issue3396;
+
+abstract class A
+{
+    abstract void M();
+}
+
+class B:A
+{
+    override void M(){ super.M(); }
+}
+
+int main()
+{
+    auto b=new B();
+    b.M();
+    return 0;
+}
```

Comment:
please remove this, if a function that uses them is required to reproduce the bug it should not be called `main`. Otherwise this looks good.

Reply:
What's the problem with it being named "main"?

Reply:
done